blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ESFieldLabel.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io;
+
+public enum ESFieldLabel {
+    metric_name,
+    tenantId,
+    unit,
+    enum_values
+}
+
```

Comment:
Note: this ESFieldLabel enum class is extracted from the ElasticIO class so that it can be used by both the existing ElasticIO class and the new EnumElasticIO class.